libsolidity/codegen/CompilerContext.h

```diff
@@ -219,6 +223,8 @@ class CompilerContext
 		OptimiserSettings const& _optimiserSettings = OptimiserSettings::none()
 	);
 
+	yul::Object& optimizeYul(yul::Object& _object, yul::EVMDialect const& _dialect, OptimiserSettings const& _optimiserSetting, std::set<yul::YulString> const& _externalIdentifiers = {});
```

Comment:
I think
```suggestion
	void optimizeYul(yul::Object& _object, yul::EVMDialect const& _dialect, OptimiserSettings const& _optimiserSetting, std::set<yul::YulString> const& _externalIdentifiers = {});
```
is clearer because otherwise you could think that a copy is returned.